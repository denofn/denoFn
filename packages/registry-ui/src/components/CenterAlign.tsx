import React from "react";
import { css } from "otion";

export function CenterAlign({ children }: React.PropsWithChildren<{}>) {
  return (
    <div
      className={css({
        width: "100%",
        marginLeft: "auto",
        marginRight: "auto",
        display: "flex",
        alignItems: "center",
        justifyContent: "center",
      })}
    >
      {children}
    </div>
  );
}
