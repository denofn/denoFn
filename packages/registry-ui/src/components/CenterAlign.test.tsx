import { render, cleanup } from "@testing-library/react";
import React from "react";

import { CenterAlign } from "./CenterAlign";

afterEach(cleanup);

it("renders with the correct styles", () => {
  const { asFragment } = render(<CenterAlign />);
  expect(asFragment()).toMatchSnapshot();
});
