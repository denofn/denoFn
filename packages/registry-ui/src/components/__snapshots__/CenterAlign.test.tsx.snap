// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with the correct styles 1`] = `
<DocumentFragment>
  <div
    class="_1d3w5wq _1o3nkn _1tx1l5v _zjik7 _1h3rtzg _f7ay7b"
  />
</DocumentFragment>
`;
